llvm/test/CodeGen/WebAssembly/int-mac-reduction-loops.ll

```diff
@@ -0,0 +1,251 @@
+; RUN: opt -mattr=+simd128 -passes=loop-vectorize %s | llc -mtriple=wasm32 -mattr=+simd128 -verify-machineinstrs -o - | FileCheck %s
```

Comment:
Why is combining vectorizer with codegen necessary to test this patch? This test would depend on scheduling while the patch in only changing TTI.

Reply:
Because as the vectorizer TTI hooks are implemented and improved, we can see how the codegen changes.